Added config and the previous code for executable detection.

MarijnStevens opened the pull request:
Replaced the `// TODO figment` statement with very basic configuration. If the executable isnt found with the hardcoded path, it will just ask the user to give an executable path.

francisdb commented:
Wow, prize for first non-me contributor goes to @MarijnStevens 👏

Did a quick review, looks good. Some questions:

* we might want to store this file in a different location? I would suggest https://crates.io/crates/dirs and create a subdirectory for vpxtool there.
  ```rust
  dirs::config_dir();
  // Lin: Some(/home/alice/.config)
  // Win: Some(C:\Users\Alice\AppData\Roaming)
  // Mac: Some(/Users/Alice/Library/Application Support)
  ```
* mind adding a small section in the readme about the config file?

MarijnStevens commented:
I've no clue what I am doing in Rust, so if something could be improved code wise just say it or change it; It can't be efficent xD. 

> * we might want to store this file in a different location?

Sure. I dont "care" really where to store the config. The default config is the obvious choice, but me personally don't like that location... To much garbage is left without an official uninstaller (and even then). But we can always make it a choice. 

> * mind adding a small section in the readme about the config file?

NP.

I've also added a .editorconfig because I use 2 space indent often. You might also want to run a `cargo fmt` as well because it wants to re-format several files (I only check-in the files I changed).
